index.html

```diff
@@ -67,30 +74,55 @@ <h2>
     <h2>
       Model
     </h2>
-    <dl class="definitions" data-sort="" data-cite="vc-data-model">
+
+    <p class="issue" title="Definitions under discussion">
+The goal of the definitions in this section is to re-use or establish
+terminology that is common across a variety of digital credential formats and
+protocols.
+    </p>
+
+    <dl class="definitions" data-sort="" data-cite="vc-data-model-2.0">
       <dt>
         <dfn>Digital credential</dfn>
       </dt>
       <dd>
-        <p>
-          Is a [=verifiable credential=] about a person.
-        </p>
+        A cryptographically secured digital document containing one or more
+        [=claims=] made by an [=issuer=] about one or more [=subjects=]. Examples include
+        ISO mobile driver's licenses [[ISO18013-5]] and W3C [=verifiable
+        credentials=] [[VC-DATA-MODEL-2.0]].
+      </dd>
+      <dt>
+        <dfn>Credential provider</dfn>
+      </dt>
+      <dd>
+        Software that provides a user interface for querying, selecting, and
+        establishing consent for the [=presentation=] of a [=digital
+        credential=]. Examples include a digital wallet, or a password manager,
+        that manages various
+        <a href="#credential-type-examples">credential types</a>.
       </dd>
       <dt>
-        <dfn>Identity credential provider</dfn>
+        <dfn>Presentation request</dfn>
       </dt>
       <dd>
-        An application or service that provides a user interface for selecting
-        and/or querying a [=digital credential=], such as a digital wallet that
-        manages various identity documents and credentials.
+        A format that a [=verifier=] uses, via an [=exchange protocol=], to
+        request a [=digital credential=] from a [=credential provider=].
       </dd>
       <dt>
-        <dfn data-for="digital credential">Request protocol</dfn>
+        <dfn>Presentation</dfn>
```

Comment:
Maybe: 

```suggestion
        <dfn>Presentation response</dfn>
```

Reply:
The term doesn't get into how its represented, but media types are one way that we could keep the flexibility in the API to be able to add different presentation formats over time.

Reply:
per comment above +1 to removing presentation,  and instead define terms `query language`, `credential format`, `exchange protocol`, which seems to be closer to the intent here.

Reply:
Yeah, "credential format" feels right here. And the format could have a MIME type or whatever... we could definitely add that. Note that the request protocol is always part of the response in the API, so you kinda know what you are going to get... however, it might be nice to have the MIME type if a query might result in multiple credential formats.

Reply:
Hrm, unfortunately, "credential format" typically only covers the format for the original credential... the original "credential format" can then be derived into another credential/presentation or wrapped (to counter-sign the original credential). "Credential format" doesn't cover the response use cases.

Reply:
"response" is also an "instance" (of specific bytes) rather than a "class" (of formats) right? that is, "response" isn't a "format", it is a "series of bytes encoded in a specific format", right?

Reply:
Correct.